src/build.sh

```diff
@@ -154,6 +161,21 @@ function kernel_build()
   return "$ret"
 }
 
+function build_clean()
+{
+  local raw_options="$*"
+  local output_kbuild_flag=''
+
+  env_name=$(get_current_env_name)
+
+  if [[ "$?" == 0 ]]; then
+    options_values['ENV_PATH_KBUILD_OUTPUT_FLAG']="${KW_CACHE_DIR}/${env_name}"
```

Comment:
I guess you don't need this line, you can set the path directly on `output_kbuild_flag`